WickedEngine/wiResourceManager.cpp

```diff
@@ -513,6 +516,90 @@ namespace wi
 					else assert(0); // failed to load DDS
 
 				}
+				else if (!ext.compare("QOI"))
```

Comment:
if possible, it would be better to share most of the code of the stb_image branch, except the way you get the pointer to the pixels. I mean by combining the two branches into one.

Reply:
true, should I just compare it inside the last else then with ext.compare() then?

Reply:
figured it out, am pushing requested changes in a min